src/panels/config/network/supervisor-network.ts

```diff
@@ -656,8 +656,10 @@ export class HassioNetwork extends LitElement {
     if (id === "address") {
       const index = (ev.target as any).index as number;
       const { mask } = parseAddress(value);
-      this._interface[version]!.address![index] = formatAddress(value, mask);
-      this.requestUpdate("_interface");
+      if (mask) {
```

Comment:
This means a change in IP address is never picked up as long as there is no mask?

Also means that if there is a re-render before the mask is updated, the changes of the user are undone.

Reply:
Ahh yep, just realised this after playing some more. It looks like adding an empty IP address adds with /24 or /64, so there's always a netmask behind the scenes, which we can use as the default instead.